### PR TITLE
ISPN-11067 Extract labels

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-11067

### CODE REVIEW

File: src/app/assets/languages/en.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "welcome-page" : {
+    "description1": "{{brand}} is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. Available as an embedded Java library or as a language-independent service accessed remotely over a variety of protocols (Hot Rod, REST)",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:3]:
```suggestion
    "description1": "{{brandname}} is a distributed in-memory key/value data store with optional schema, available under the Apache License 2.0. Available as an embedded Java library or as a language-independent service accessed remotely over a variety of protocols (Hot Rod, REST)",
```

File: src/app/assets/languages/en.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "description2": "Use {{brand}} as a cache or data grid in containerized, cloud-native environments or on bare-metal. Works on AWS, Azure, Google Cloud, Kubernetes and OpenShift.",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:4]:
```suggestion
    "description2": "Use {{brandname}} as a cache or data grid in containerized, cloud-native environments or on bare-metal. Works on AWS, Azure, Google Cloud, Kubernetes and OpenShift.",
```

File: src/app/assets/languages/en.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "connect" : "Remotely connect to {{brand}} from client applications.",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:5]:
```suggestion
    "connect" : "Remotely connect to {{brandname}} from client applications.",
```

File: src/app/assets/languages/en.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "download" : "Download a client",
+    "servers" :  "Find out how to set up, configure, and monitor {{brand}} servers.",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:7]:
```suggestion
    "servers" :  "Find out how to set up, configure, and monitor {{brandname}} servers.",
```

File: src/app/assets/languages/en.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    "learn-more": "Learn more",
+    "develop": "Develop applications to use {{brand}} for in-memory storage in a variety of use cases.",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:9]:
```suggestion
    "develop": "Develop applications to use {{brandname}} for in-memory storage in a variety of use cases.",
```

**Comment by karesti** [comment]
@oraNod should work

**Comment by oraNod** [comment]
@karesti still not getting the console to load. when I run the console it shows empty pages. I've been looking for errors and trying to debug. so far I've found "Uncaught ReferenceError: availableLocales is not defined" but not much else. on Fedora 32 and have tried firefox and chrome.

**Comment by karesti** [comment]
@oraNod have you done a clean up and npm install and so ?

**Comment by oraNod** [comment]
@karesti yeah I've done it all. just double checked again this morning. if I switch over to the master branch and build the console it works as expected. then I switch back to this branch and all the pages are empty. specifically, if I go to "http://localhost:9000/console/" or "http://localhost:9000/console/container/" or "http://localhost:9000/console/welcome" it seems like the console is loading something but all that appears in the browser is an empty/white page.

Here are some messages I got from the debugger: https://paste.centos.org/view/fe63ce20 

Let me know if there's anything else I can provide.

**Comment by karesti** [comment]
@oraNod please retry

**Comment by oraNod** [comment]
@karesti it's fixed! works fine and I can modify the strings now.

**Comment by karesti** [comment]
let's work here to externalize labels
I will rebase master myself while PR get integrated so don't bother about that
If you have troubles about pluralization, let me know because maybe something is missing about that

**Comment by oraNod** [comment]
@karesti hey! I've made some progress and have a start on the strings. I'll keep working through but wanted to push a commit so you could take a look at the changes I've made so far. let me know if it looks like I'm not on the right track with the approach. cheers.

**Comment by karesti** [comment]
@oraNod rebased

**Comment by karesti** [comment]
@oraNod grab a look if everything seems ok, and then I will merge

**Comment by oraNod** [comment]
@karesti yes, I think this is good to merge. we don't have to extract all the labels in one PR. that'll start to be a rebase nightmare. this is a good start. from here we can add new strings via the properties file. any other strings we can externalize "lazily" as we go through review cycles. 

I also opened ISPN-11412 and will work on those baseline edits from the first review session.

**Review by oraNod — APPROVED** [review]

**Pull request merged**